examples/scripts/README.rst

```diff
@@ -222,8 +222,6 @@ PDHG
 
     `ct_svmbir_tv_multi.py <ct_svmbir_tv_multi.py>`_
        TV-Regularized CT Reconstruction (Multiple Algorithms)
-    `denoise_tv_multi.py <denoise_tv_multi.py>`_
```

Comment:
Why was this removed? Merge error, perhaps?

Reply:
Good catch. I did not notice this during merge. Corrected.